telefunc/node/server/shield/codegen/transformer.ts

```diff
@@ -0,0 +1,97 @@
+export { shieldTelefunctions }
+
+import { Project, VariableDeclarationKind } from 'ts-morph'
+
+function parseEmitMessages(messages: string[]) {
+  const matcher = /Type '"([A-Za-z0-9_]+)"' is not assignable to type '"([^"]+)"'./
+  const teleFuncToShieldStr: Record<string, string> = {}
+  
+  for (let message of messages) {
+    const found = message.match(matcher)
+    if (!found) {
+      console.warn("Encountered unrecognized emit message: ", message)
```

Comment:
While this error message and the one below are convenient for us, they are confusing for users. We should use `assert()` instead. We can still add some debug info if we want. Example of using `assert()` with some debug info:

https://github.com/vikejs/telefunc/blob/4aa22608b5be742a6b1c5ff78c8535a41aa32547/telefunc/node/vite/loadTelefuncFilesWithVite.ts#L22

Reply:
Hmm, I don't want to stop on failure. Just a warning would suffice.